lib/aytests/test_runner.rb

```diff
@@ -90,7 +90,15 @@ def build
     # @return [String]      AutoYaST profile path for the given stage
     def autoinst(stage = :install)
       autoinst = tests_path.join("#{test_name}.#{stage}_xml")
-      autoinst.file? ? autoinst : tests_path.join("#{test_name}.xml")
+      unless autoinst.file?
```

Comment:
What about returning to avoid indentation? Something like this...

```ruby
return autoinst if autoinst.file?
autoinst = tests_path.join(test_name)
return autoinst if autoinst.directory?
tests_path.join("#{test_name}.xml")
```